### ISSUE TITLE
Add a way to mark an idea that has already been implemented

### ISSUE THREAD

**Issue by conscho** [cornelltech/unicornucopia-issue-tracker#91]
It seems like there are many ideas in unicornucopia that have been implemented already. It would be helpful to mark those ideas as not "new". So people who posted these ideas should explain how to differentiate from the existing solutions.

**Comment by everydayslang** [comment]
When you say "not new" you mean there is already a company in this space? I ask because might a link to the company be more benificial? Or is it something that you'd like to be revealed in the list view so you could quickly scan for this information while browsing?

**Comment by conscho** [comment]
Exactly. New as in there is already a company in this space. I'm not sure how to best include this information. But a link to the existing company would be a good start.

**Comment by AVatch** [comment]
I think "comparable" is a good word to describe this as just because there is someone doing something should not discourage one from pursuing the opportunity.
That being said, I am hesitant as this sounds like it overfits the use case for startup ideas rather than ideas in general.

On Sun, Oct 16, 2016 at 12:46 PM -0400, "Constantin Scholl" notifications@github.com wrote:

Exactly. New as in there is already a company in this space. I'm not sure how to best include this information. But a link to the existing company would be a good start.

##
